mmdet/models/dense_heads/lad_head.py

```diff
@@ -0,0 +1,239 @@
+import torch
+from mmcv.runner import force_fp32
+
+from mmdet.core import bbox_overlaps, multi_apply
+from ..builder import HEADS
+from .paa_head import PAAHead, levels_to_images
+
+
+@HEADS.register_module()
+class LADHead(PAAHead):
+    """Label Assignment Head from the paper: `Improving Object Detection by
+    Label Assignment Distillation <https://arxiv.org/pdf/2108.10520.pdf>`_"""
+
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds', 'iou_preds'))
+    def get_la(self,
```

Comment:
You can use the full name directly, such as `label_assignment_distillation()`.

Reply:
Thanks, I will change the name to `get_label_assignment`.